### PR TITLE
Create dashboards.md

### PR DESCRIPTION
**Pull request by patski123**

### CODE REVIEW

File: docs/self-hosted-apps/dashboards.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Self Hosted Dashboards
+
+As always, check out [awesome-selfhosted](https://github.com/awesome-selfhosted/awesome-selfhosted) but here are our favourites.
+
+## Heimdall
+
```
Comment (WhiteNight121) [inline docs/self-hosted-apps/dashboards.md:6]: you might just want to add a description like you did below for organizr.

File: docs/self-hosted-apps/dashboards.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# Self Hosted Dashboards
+
```
Comment (py-crash) [inline docs/self-hosted-apps/dashboards.md:2]: Add a short description about what you mean by "dashboards". Is [netdata](https://www.netdata.cloud/) for e.g. a dashboard?

**Comment by BCNelson** [comment]
This is looking dangerously close to just being a list of software. More details are going to be needed before it's merged.

**Comment by RealOrangeOne** [comment]
> This is looking dangerously close to just being a list of software. More details are going to be needed before it's merged.

Definitely agree. The "Reviews and Related Links" sections are definitely useful. But it's exactly content like that the wiki is for, rather than simply enumerating options.

**Comment by WhiteNight121** [comment]
maybe try and compare 3-4 of them pro/cons ect

**Comment by IronicBadger** [comment]
Closing due to lack of activity. Please note we have changed to `master` as the default branch for future PRs. This article will be very useful so please consider opening a new PR with the comments from this one incorporated.

Thanks!

**Pull request closed without merging**